ts/mixins/legend-symbol.ts

```diff
@@ -138,12 +138,8 @@ H.LegendSymbolMixin = {
 
         this.legendLine = renderer
             .path([
-                'M',
-                0,
-                verticalCenter,
-                'L',
-                symbolWidth,
-                verticalCenter
+                ['M', 0, verticalCenter],
+                ['L', symbolWidth, verticalCenter]
```

Comment:
Is this from a different PR?

Reply:
Unrelated bug in code outside my branch (I added a fix here). The point is that some time ago it happened that paths have to be passed as segments and it is enough to fix the code for something like that otherwise it causes problems.